### ISSUE TITLE
Error for profile 19041

### ISSUE THREAD

**Issue by swepeba** [tomchop/volatility-autoruns#30]
Hi,

I get an error when running this plugin with profile 19041 (Volatility 2.6.1).
Nothing more is shown when adding `-v`

`ERROR   : volatility.debug    : Unable to find registry hives.`

It works for older profiles.
Any ideas?

**Comment by jared703** [comment]
Are you seeing any hives resident in memory via hivelist?

**Comment by swepeba** [comment]
No, both hivelist and hivescan are empty...
Maybe something wrong in the profile from the Volatility project, because when using Volatility3 and windows.registry.hivelist.HiveList on the same memory dump it works fine.

**Comment by jared703** [comment]
starting from the correct profile is critical. Is everything else normal
(pslist, pscan, etc.)?

**Comment by swepeba** [comment]
Yes. All the other plugins work with the same profile. Since hivelist and hivescan do not work either I suppose it might be some problems inside the 19041 profile definition.

**Comment by jared703** [comment]
It could be, especially in vol3 is detecting registry data.
